docs/csharp/quick-starts/list-collection.yml

```diff
@@ -162,7 +162,7 @@ items:
     > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
     
 - content: |
-    You've completed the list quick start. This quick start is the final interactive quick start. You can continue these quickstarts on your own development environment. The .NET topic [Get Started in 10 minutes](https://www.microsoft.com/net/core) has instructions for setting up your local development environment. Once you've done that, continue with [Introduction to classes](introduction-to-classes.md) in your local environment.
+    You've completed the list quick start. This quick start is the final interactive quick start.     You can continue these quick starts on your own development environment. Learn the basics of [local development](local-environment.md) and then pick a quick start. You can try this same exercise, move directly to the next quick start, or start again at with the [numbers in C#](numbers-in-csharp-local.md) quickstart.
```

Comment:
Is there a reason for the spaces inserted between "start." and "You"? Did you deliberately want the sentence to start on a new line?

Reply:
I thought it should be part of the same paragraph. I removed the extra spaces.